### DISCUSSION TITLE
Automation in home assistant

### DISCUSSION THREAD

**Discussion by MilanAra88** [PRProd/HA-Firemote#342] [General]
Hi there, 

Thank you so much for this firemote - really useful!!

I was just wondering how i could use this in a automation in home assistant? Based on a trigger, I would like to switch on the firestick, and then hit a quick app launch button. 

1) Trigger : for me its to play music on server (volumio)
2) Action: Press home button on fire stick to switch on fire stick and to switch on TV --> then press quick app button (which is just a silk web page for the music server) 

I saw the post about ADB scripts but I have no experience with this.  How can they be configured in the home assistant config YAML file? 

Bit of a noob :) apologies
 
---

Just looking the the ADB automation post - do i just add the ADB commands into the action below?
<img width="1230" alt="image" src="https://github.com/PRProd/HA-Firemote/assets/43094647/b92eae1a-f3b2-4d49-b039-665e9ebf7653">

-----

ok so the above work :) - again thank you for this card! amazing! 

---

one question i do have is I need to find the "Bookmark-1 " app command - following the ADB post - i cant seem to see the commands appear in the console in chrome when in developer mode. Any advise on how to do this? 

-- ok scratch all the above :) I found it in the code you provided in the post - adb shell am start -n com.esaba.bookmarker1/.MainActivity

Feel free to delete this thread :)

**Comment by PRProd** [comment]
Greetings @MilanAra88!

I had a huge smile on my face when I read your post today.  It reminded me of myself.  I always seem to find the answer to my question right after I send a message to someone asking for help.  :smile:

Learning can be frustrating sometimes, but also rewarding.  I'm glad that you were able to find all of your answers!

Feel free to reach out with any questions you might have in the future!  Also, congratulations!  You're no longer a noob.  :smile:
